### PR TITLE
Feature/delete dse projects

### PR DESCRIPTION
**Pull request by lukasmweber**
Using the newly introduced CL-Option `--deleteProjects`, Project Files can now be deleted after compose or explore runs.
The option also allows overriding to keep projects using these variants:
`--deleteProjects true` -> delete all Project Files
`--deleteProjects false` -> keep all Project Files
`--deleteProjects` -> delete all Project files (equal to `--deleteProjects true`)

### CODE REVIEW

File: src/main/scala/tapasco/task/ComposeTask.scala [hunk 1]
```diff
@@ -58,6 +59,7 @@ class ComposeTask(composition: Composition,
   def job: Boolean = if (! _slurm) nodeExecution else slurmExecution
 
   private def nodeExecution: Boolean = {
+
```
Comment (sommerlukas) [inline src/main/scala/tapasco/task/ComposeTask.scala]: Unrelated whitespace change;

File: src/main/scala/tapasco/parser/Usage.scala [hunk 2]
```diff
@@ -163,6 +163,7 @@ configuration via `tapasco -n config.json`.
                                         """default: "Vivado"""") &
            Arg("--features FEATURES", "configures Features, see `tapasco -h features`" &
                                       "syntax: FEATURE [, FEATURE]*") &
+           Arg("--deleteProjects (true | false)?", "Spefify whether project files are deleted or kept") &
```
Comment (sommerlukas) [inline src/main/scala/tapasco/parser/Usage.scala]: Please document the default value of the boolean flag;

File: src/main/scala/tapasco/parser/Usage.scala [hunk 3]
```diff
@@ -317,6 +318,7 @@ configuration via `tapasco -n config.json`.
                                   "default: number of CPUs") &
            Arg("--debugMode NAME", "dry run, no compositions are executed, see" ~
                                    "`tapasco -h compose`") &
+           Arg("--deleteProjects ( true | false )?", "Spefify whether project files are deleted or kept") &
```
Comment (sommerlukas) [inline src/main/scala/tapasco/parser/Usage.scala]: Same as above: Please document the default value of the boolean flag;

**Comment by lukasmweber** [comment]
All requested Changes were made accordingly.
